chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -210,8 +210,10 @@ rtfScan-output-file = Output File
 zotero-file-none-selected =
     .value = No file selected
 
-zotero-file-choose =
-    .label = Choose File…
+rtfScan-file-choose-input =
+    .label = Choose { rtfScan-input-file }
+rtfScan-file-choose-output =
+    .label = Choose { rtfScan-output-file }
```

Comment:
I don't think we want to use variables for something like this. This is just a normal phrase, and the words might be different in the different contexts. We'd only use a variable if we were referring to, say, an actual button label that appears elsewhere.